assets/default-keybindings.json

```diff
@@ -23,6 +23,54 @@
       "command": "generic:deleteItem",
       "keys": ["Backspace", "Delete"]
     },
+    {
+      "command": "generic:space",
+      "keys": ["Space"]
+    },
+    {
+      "command": "generic:tab",
```

Comment:
some big picture thoughts, does it make sense to limit keybindings to keyDescriptions? I think for now it does

I guess for plugins, we would have the plugin include a keybindings file? I think that makes sense 
Like you could download a plugin and it would be like a folder zip with .tsx files and a keybindings.json or maybe a <any-prefix>keybindings.json 
And we would scan our plugins folder, find all of these keybindings files and integrate them into our keybindings map? 
And we would "force" plugins to declare their keybindings in this way

I think that makes sense

We can't really force plugins to use our keybindings api currently, if we let them give us react elements I don't know a way that we can stop them from using onKeyDown, but I think that's fine I don't think we're really concerned with malicious plugins, it's more that plugins would want to declare their keybindings through our api so that they show up in the command palette and people can rebind them

Reply:
I also wonder if this is limiting for us in the future
I guess not, and I guess if we come across a future case where we really want to use a keybinding that doesn't make sense to just throw on the default-keybindings.json list, it is very easy to change our system to allow registering key descriptions directly. I don't think we will end up in a situation where we feel limited by this design choice, I think that the system is flexible enough